### PR TITLE
Mvc cleanup

### PR DESCRIPTION
**Pull request by marcincichon**
@ollietreend @RobertWDLowe can you review?

Contains:
about us
mvc cleanup
moving nested menu functionality to theme

### CODE REVIEW

File: wp-content/themes/mojintranet/models/menu.php [hunk 1]
```diff
@@ -1,38 +1,62 @@
...
     }
 
     $organised_menu_items = $this->_build_menu_tree_recursive($menu_items, $depth_limit);
 
-    return array(
-      'results' => $organised_menu_items
-    );
+    return [
+      'results' => $organised_menu_items,
+      'top_level_id' => $top_level_id
```
Comment (RobertWDLowe) [inline wp-content/themes/mojintranet/models/menu.php]: $top_level_id - not using $this and what is it used for.

File: wp-content/themes/mojintranet/modules/api/page_tree.php [hunk 2]
```diff
@@ -33,14 +37,24 @@ protected function route() {
...
-      'agency' => $params[1],
-      'additional_params' => $params[2],
-      'page_id' => (int) $params[3],
-      'depth' => (int) $params[4],
-      'order' => $params[5]
+      'method' => $method,
     );
+
+    if($method === 'children-by-tag') {
+      $this->params['tag'] = $params[1];
```
Comment (RobertWDLowe) [inline wp-content/themes/mojintranet/modules/api/page_tree.php]: why no agency???

File: wp-content/themes/mojintranet/views/modules/header.php [hunk 3]
```diff
@@ -50,7 +50,7 @@
   <div class="header-menu">
     <div class="grid">
       <div class="col-lg-12 col-md-12 col-sm-12">
-        <?php dynamic_sidebar('main-menu') ?>
+        <?php $this->view('modules/main_menu', ['main_menu' => $main_menu]) ?>
```
Comment (RobertWDLowe) [inline wp-content/themes/mojintranet/views/modules/header.php]: delete  main menu sidebar

File: wp-content/themes/mojintranet/views/pages/about_us/main.php [hunk 4]
```diff
@@ -8,28 +8,15 @@
...
+      <!--<div class="excerpt hq-only">
         <?=$excerpt?>
-      </div>
-      <p class="agency-only">
-        <a class="toggle-global-categories-box" href="#">
-          <span class="expand-icon icon"></span>
-          Learn more about the MoJ
-        </a>
-      </p>
+      </div>-->
```
Comment (RobertWDLowe) [inline wp-content/themes/mojintranet/views/pages/about_us/main.php]: remove commented out

**Comment by ollietreend** [comment]
Looks good @marcincichon 👍

**Pull request merged**